opensrp-reveal/src/test/java/org/smartregister/reveal/view/ListTasksActivityTest.java

```diff
@@ -796,4 +799,21 @@ public void testSetSearchPhrase() {
     }
 
 
+    @Test
+    public void testDisplayResetTaskInfoDialog() {
+        Whitebox.setInternalState(listTasksActivity, "listTaskPresenter", listTaskPresenter);
+        listTasksActivity.displayResetInterventionTaskDialog(BEDNET_DISTRIBUTION);
+
+        AlertDialog alertDialog = (AlertDialog) ShadowAlertDialog.getLatestDialog();
+        assertTrue(alertDialog.isShowing());
+
+        TextView tv = alertDialog.findViewById(android.R.id.message);
+        assertEquals(getString(R.string.undo_task_msg), tv.getText());
+
+        alertDialog.getButton(DialogInterface.BUTTON_POSITIVE).performClick();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'DialogInterface.BUTTON_POSITIVE' due to existing static import 'android.content.DialogInterface.BUTTON_POSITIVE'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4997357)